.openshift-ci/drivers/scripts/kernel-splitter.py

```diff
@@ -23,16 +23,25 @@ def parse(task_line: str):
 
         return Task(task[0], task[1], task[2])
 
+    def is_ebpf(self):
+        return self.driver_type == "bpf"
+
+    def is_kernel_module(self):
+        return self.driver_type == "mod"
+
 
 class Builder:
-    def __init__(self, name, regex, tasks):
+    def __init__(self, name, regex, tasks, ebpf_only=False, ko_only=False):
```

Comment:
Might be worth it to replace these 2 booleans with an enum, I think it could improve readability, we could have something like this:
```py
from enum import Enum

class DriverType(Enum):
    EBPF_ONLY = 0,
    KMOD_ONLY = 1,
    ALL_DRIVERS = 2,

class Builder:
    def __init__(self, name, regex, tasks, driver_type = DriverType.ALL_DRIVERS):
```
Or if you want to really piss off people, you can have the enum be:
```py
class DriverType(Enum):
    EBPF_ONLY = 0x1,
    KMOD_ONLY = 0x2,
    ALL_DRIVERS = 0x3,
```
And handle checks with bitwise operators.